src/main/scala/com/gu/invoicing/common/ZuoraAuth.scala

```diff
@@ -46,28 +49,36 @@ object ZuoraAuth extends JsonSupport {
       )
       .asString
       .body
+      .tap { body =>
+        // although cloudwatch is pretty secure, it's a nice thing to redact if possible
+        val redactedBody = body.replaceAll(""""access_token":"[^"]*"""", """"access_token":"***REDACTED***"""")
+        logger.info(redactedBody)
+      }
       .pipe(read[AccessToken](_))
       .access_token
   }
-  private val timer = new Timer()
+  private val timer = new Timer(true)
```

Comment:
makes this timer a daemon, meaning it doesn't keep the JVM running when the main thread has terminated
https://docs.oracle.com/javase/8/docs/api/java/util/Timer.html#Timer-boolean-